packages/command-palette/src/index.tsx

```diff
@@ -208,20 +230,28 @@ interface NotFoundMessageProps {
 	currentRoute: string | null;
 }
 
+interface CommandPaletteProps {
+	currentSiteId: number | null;
+	navigate: ( path: string, openInNewTab: boolean ) => void;
+	useCommands: ( options: useCommandsParams ) => PaletteCommand[];
+	wpcom: WPCOM;
+	currentRoute: string | null;
+	singleSiteMode: boolean;
```

Comment:
This prop is no longer needed

Reply:
https://github.com/Automattic/wp-calypso/pull/87790/commits/470112d6bbd3091301ee8ea65b3b825a819d4b60